### ISSUE TITLE
Color Contrast: Integration test failures in IE10 depending on browser window size

### ISSUE THREAD

**Issue by dsturley** [dequelabs/axe-core#120]
IE uses floats for things in `commons.dom.visuallyContains`; causing imprecision that will cause tests to fail depending on the browser window size.  Specifically the `bottom` scroll area calc is off by about `0.000001`

**Comment by WilcoFiers** [comment]
While working on this issue I noticed a similar issue with Chrome, where the 'right' value was off by about .000004. I'm guessing we can see more such things pop up from time to time so I think adding a small margin is probably the way to go here.

**Comment by WilcoFiers** [comment]
Issue has been resolved and the fix will be available in the next major release.
